experimental/devices/rainbowhat/rainbowhat.go

```diff
@@ -0,0 +1,184 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+package rainbowhat
+
+import (
+	"periph.io/x/periph/conn/spi/spireg"
+	"periph.io/x/periph/devices/apa102"
+	"periph.io/x/periph/devices/bmxx80"
+	"periph.io/x/periph/experimental/devices/ht16k33"
+	"periph.io/x/periph/host/rpi"
+
+	"periph.io/x/periph/conn/gpio"
+	"periph.io/x/periph/conn/i2c/i2creg"
+)
+
+const (
+	numPixels  = 7
```

Comment:
Inline these two, no need for a constant since it won't change and they are used exactly once.